extras/videoDrivers/SDL2/VideoSDL2.cpp

```diff
@@ -223,6 +223,12 @@ void VideoSDL2::PrintError(const std::string& msg) const
     boost::nowide::cerr << msg << std::endl;
 }
 
+void VideoSDL2::ShowErrorMessage(const std::string& title, const std::string& message)
+{
+    // window==nullptr is okay too ("no parent")
```

Comment:
I would have left out this comment as it only repeats the documentation of the called function and if we did that more often the code would become harder to read.   
But whatever...

Reply:
added a comment